package.json

```diff
@@ -46,13 +46,13 @@
     "@storybook/addon-a11y": "3.4.11",
     "@storybook/addon-actions": "3.4.11",
     "@storybook/addon-backgrounds": "3.4.11",
-    "@storybook/addon-info": "3.4.11",
+    "@storybook/addon-info": "^5.1.9",
```

Comment:
You definitely wont want to making any changes to dependencies in this PR...

Reply:
I don't remember changing this on my own. Maybe it occurred when I updated snapshot tests? Do you just want me to switch it back to the old version and push it back up?

Reply:
https://github.com/OperationCode/front-end/pull/640/commits/5754ca8871396a939319f916e9cd24e8f6838362

It happened in this commit.

Reply:
changed it back :)

Reply:
There were other dependencies in there which were updated unnecessarily. I've reverted those changes. Be careful with `package.json` and `yarn.lock` - there shouldn't be a need to touch those files on this PR going forward.